src/pytest/test_gc.py

```diff
@@ -26,7 +26,9 @@ def testBasicGC(self):
 
         self.assertEqual(self.cmd('ft.del', 'idx', 'doc2'), 1)
 
-        time.sleep(1)
+        for i in range(100):
```

Comment:
for _ is the normal python convention if you are not using the iterator variable.